src/Pos/Generator/Block/Mode.hs

```diff
@@ -167,6 +180,18 @@ mkBlockGenContext bgcParams@BlockGenParams{..} = do
         bgcTxpMem <- (,ignoreTxpMetrics) <$> mkTxpLocalData
         bgcDelegation <- mkDelegationVar @SscGodTossing
         return BlockGenContext {..}
+  where
+    -- Genesis utxo is needed only for boot era stakeholders
+    bgcGenesisUtxo =
+        let addrs =
+                -- So we take three stakeholders in boot era.
+                take 3 $
+                map (makePubKeyAddress . toPublic) . toList $
+                view (bgpSecrets . asSecretKeys) bgcParams
+            utxoTxHash = unsafeHash ("randomutxotx" :: Text)
+            txIns = map (TxIn utxoTxHash) [0..fromIntegral (length addrs) - 1]
+            txOuts = map (\addr -> TxOutAux (TxOut addr (mkCoin 10000)) []) addrs
+        in GenesisUtxo $ M.fromList $ txIns `zip` txOuts
```

Comment:
It looks totally weird, but let's deal with it after merging this PR. I don't like that here you recreate genesis utxo which differs from the real genesis utxo.